xwiki-platform-core/xwiki-platform-like/xwiki-platform-like-test/xwiki-platform-like-test-docker/src/test/it/org/xwiki/like/test/ui/LikeIT.java

```diff
@@ -129,4 +132,28 @@ void likeUnlikeDefaultConfiguration(TestUtils testUtils, TestReference testRefer
         assertTrue(likeButton.isDisplayed());
         assertEquals(1, likeButton.getLikeNumber());
     }
+
+    @Test
+    @Order(3)
+    void userProfileUIXLiveData(TestUtils testUtils, TestReference testReference)
+    {
+        testUtils.login(USER1, USER1);
+
+        // Re-creates and likes a page.
+        testUtils.deletePage(testReference);
+        testUtils.createPage(testReference, "", "");
+        LikeButton likeButton = new LikeButton();
+        likeButton.clickToLike();
+        assertEquals(1, likeButton.getLikeNumber());
+
+        // Go to the Like Pages user profile tab and assert that the like page is correctly displayed in the live data.
+        UserProfileLikePagesPage userProfileLikePagesPage = new UserProfileLikePagesPage(USER1);
+        userProfileLikePagesPage.gotoPage();
+        LiveDataElement likedPages = userProfileLikePagesPage.getLiveData();
+        likedPages.waitUntilHasContentReady();
+        assertEquals(1, likedPages.countRows());
+        assertTrue(likedPages.hasLinkRow(TITLE_COLUMN_NAME, testReference.getLastSpaceReference().getName(),
+            testUtils.getURL(testReference.getLastSpaceReference())));
+        assertTrue(likedPages.hasRow(LIKES_COLUMN_NAME, "1"));
+    }
```

Comment:
Generally speaking I don't understand this test method. For me it's part of testing the like feature:
- you like a page somewhere
- you then go to the use profile and navigate to the like tab and verify it's displayed

Reply:
The way you've written it is clearly as a unit test to me and not as a functional test (i.e. scenario). I've seen that in other tests you added in the past but I didn't report it back then.

Reply:
I'm in addition testing that the displayed content is the one expected (the right liked page with the right likes number). If that's over-testing I'm ok to remove these assertions.

Reply:
That seems good